### PR TITLE
Fixes 4851: remove pulp-web

### PR DESCRIPTION
**Pull request by jlsherrill**
## Summary

Removes pulp-web from docker-compose  as its not used in our real deployments and we no longer need it.

as part of this, the pulp-api is still available on :8080 but pulp-content is now available on :8081
This caused some tests to need updates that were assuming api and content were on the same port

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMS-4851

**Review by dominikvagner — COMMENTED** [review]
looks good! 👍🏼 just one small nit 😅

File: compose_files/pulp/docker-compose.yml [hunk 1]
```diff
@@ -99,29 +103,13 @@ services:
...
-    command: ["/usr/bin/nginx.sh"]
-    depends_on:
-      migration_service:
-        condition: service_completed_successfully
-    ports:
-      - "8080:8080"
-    hostname: pulp
-    user: root
-    volumes:
-      - "./assets/bin/nginx.sh:/usr/bin/nginx.sh:Z"
```
[the remark this answers is not in the document]
Comment (jlsherrill) [reply compose_files/pulp/docker-compose.yml:119]: ah yes, good point!

**Review by dominikvagner — APPROVED** [review]
good job! approved!

**Pull request merged**